TaskManagement/settings.py

```diff
@@ -0,0 +1,132 @@
+"""
+Django settings for TaskManagement project.
+
+Generated by 'django-admin startproject' using Django 4.2.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-x2rpfqcus!&!%@a#q#x%m%d7kr6#aa05!=8(xtzy834!(q_sv!'
```

Comment:
we never publish secret keys like these, for all secret keys storage we use ```.env``` file.